lib/src/beamer_back_button_dispatcher.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:flutter/material.dart';
+
+import 'beamer_router_delegate.dart';
+
+class BeamerBackButtonDispatcher extends RootBackButtonDispatcher {
```

Comment:
Just a little bit of documentation on top of that class... Like; "it will do beamBack if possible"

Reply:
ok it just I am a little on a tight schedule, that I really rushed this pr

Reply:
no problem. No need to hurry with beamer

Reply:
I will definitely merge it after the comment